update-compose.yaml

```diff
@@ -0,0 +1,12 @@
+policies:
```

Comment:
Does this file need to be at the top-level?

... after some digging, I gather that it does (per https://www.updatecli.io/docs/core/compose/).
IMO, it is unfortunate that the tool is called "updatecli" but the config file just has "update" in it, which is ambiguous.

Could we please have a comment at the top of the file giving context. The file "update-compose.yaml" is definitely not a well-known file name. Something like:

```suggestion
# Config file for `updatecli compose ...`.
# https://www.updatecli.io/docs/core/compose/
policies:
```

Reply:
> Does this file need to be at the top-level?

I tried to use a different location but it didn't handle well certain configurations, see https://github.com/updatecli/updatecli/issues/2266

I guess, it will eventually support it.